From 9eff90b9cdd1d6ed1d6843cb7c6c800236ee762e Mon Sep 17 00:00:00 2001
From: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Date: Wed, 26 Oct 2016 14:14:45 -0200
Subject: [PATCH 2894/2912] driver-api: create an edac.rst file with EDAC
 documentation

Currently, there's no device driver documentation for the EDAC
subsystem at the driver-api book. Fill in the blanks for the
structures and functions that misses documentation, uniform
the word on the existing ones, and add a new edac.rst file at
driver-api, in order to document the EDAC subsystem.

Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Signed-off-by: Sudheesh Mavila <sudheesh.mavila@amd.com>
---
 drivers/edac/edac_mc.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/edac/edac_mc.h b/drivers/edac/edac_mc.h
index 97ee6a9..dcc2c7e 100644
--- a/drivers/edac/edac_mc.h
+++ b/drivers/edac/edac_mc.h
@@ -152,8 +152,6 @@ extern void edac_mc_free(struct mem_ctl_info *mci);
  *
  * If found, return a pointer to the structure.
  * Else return NULL.
- *
- * Caller must hold mem_ctls_mutex.
  */
 extern struct mem_ctl_info *edac_mc_find(int idx);
 
-- 
2.7.4

